docs/serializer-formatter.rst

```diff
@@ -68,6 +68,8 @@ to Kafka. A ``SerializationException`` may occur during the send call, if the da
     ProducerRecord<Object, Object> record = new ProducerRecord<>("topic1", key, avroRecord);
     try {
       producer.send(record);
+      producer.flush()
```

Comment:
these should probably  be in a `finally` block instead of in the `try` part. But I think we also want to be careful here -- normally you don't send a single message and close the producer, so this code could be a bit misleading as an example. And even if they don't close, we kind of don't want to imply that you should `flush` after every `send`. I would move these into a `finally` block after the `catch` and add a comment, something like

```
// When you're finished producing records, you can flush the producer to ensure it has all been written to Kafka and then close the producer to free its resources.
```